### ISSUE TITLE
All examples have same warning at the ".on", ".formValidation" using lint

### ISSUE THREAD

**Issue by gstroud** [formvalidation/support#46]
$(document).ready(function() {
 44      $('#productForm')
 45          // IMPORTANT: You must declare .on('init.field.bv')
 46          // before calling .formValidation(options)
 47          .on('init.field.fv', function(e, data) {
     ========^
     lint warning: unexpected end of line; it is ambiguous whether these lines are part of the same statement

What does the syntax need to be to avoid this warning?  My code mimics the demos and I want to find a bug with an event not firing on a Telerik autocomplete widget, so I want to eliminate this issue.

Thanks

Greg

**Comment by nghuuphuoc** [comment]
What's tool you're using for checking the warning?
I've just tried to use [JSLint](http://jslint.com/) to check the following sample code

```javascript
$('#productForm')
    // IMPORTANT: You must declare .on('init.field.bv')
    // before calling .bootstrapValidator(options)
    .on('init.field.fv', function(e, data) {
        // data.fv      --> The FormValidation instance
        // data.field   --> The field name
        // data.element --> The field element

        var $icon      = data.element.data('fv.icon'),
            options    = data.fv.getOptions(),                      // Entire options
            validators = data.fv.getOptions(data.field).validators; // The field validators

        if (validators.notEmpty && options.icon && options.icon.required) {
            // The field uses notEmpty validator
            // Add required icon
            $icon.addClass(options.icon.required).show();
        }
    })
    .formValidation({
        framework: 'bootstrap',
        icon: {
            required: 'fa fa-asterisk',
            valid: 'fa fa-check',
            invalid: 'fa fa-times',
            validating: 'fa fa-refresh'
        },
        fields: {
            name: {
                validators: {
                    notEmpty: {
                        message: 'The name is required'
                    }
                }
            },
            description: {
                validators: {
                    stringLength: {
                        max: 300,
                        message: 'The description must be less than 300 characters long'
                    }
                }
            },
            price: {
                validators: {
                    notEmpty: {
                        message: 'The price is required'
                    },
                    numeric: {
                        message: 'The price must be a number'
                    }
                }
            },
            quantity: {
                validators: {
                    notEmpty: {
                        message: 'The quantity is required'
                    },
                    integer: {
                        message: 'The quantity must be a number'
                    }
                }
            }
        }
    })
    .on('status.field.fv', function(e, data) {
        // Remove the required icon when the field updates its status
        var $icon      = data.element.data('fv.icon'),
            options    = data.fv.getOptions(),                      // Entire options
            validators = data.fv.getOptions(data.field).validators; // The field validators

        if (validators.notEmpty && options.icon && options.icon.required) {
            $icon.removeClass(options.icon.required).addClass('fa');
        }
    });
```

and didn't see the ```unexpected end of line``` warning.

**Comment by gstroud** [comment]
Hi Phuoc

Here: http://www.javascriptlint.com/online_lint.php

**Comment by nghuuphuoc** [comment]
You can avoid this warning by moving the dot (.) to the end of previous line.

Before:

```javascript
$('#productForm')
    .on('init.field.fv', function(e, data) {
        ...
    })
    .formValidation({
        ...
    })
    .on('status.field.fv', function(e, data) {
        ...
    });
```

After:

```javascript
$('#productForm').    // <--- Move the dot (.) to the end of line
    on('init.field.fv', function(e, data) {
        ...
    }).
    formValidation({
        ...
    }).
    on('status.field.fv', function(e, data) {
        ...
    });
```

These references explain how the original is warned and how it's fixed:

* https://mpdaugherty.wordpress.com/2010/06/10/jslints-ambiguous-end-of-line-vs-jquery-style/
* http://stackoverflow.com/questions/3004496/is-block-style-really-this-important

**Comment by gstroud** [comment]
Ok, thank you, sorry I didn't try that.
